src/nurse_ussd.js

```diff
@@ -175,16 +176,21 @@ go.app = function() {
     // INITIAL STATES
 
         self.add('st_subscribed', function(name) {
-            return new ChoiceState(name, {
+            return new PaginatedChoiceState(name, {
                 question: $("Welcome to NurseConnect"),
                 choices: [
                     new Choice('st_subscribe_other', $('Subscribe a friend')),
                     new Choice('st_change_num', $('Change your no.')),
                     new Choice('st_change_faccode', $('Change facility code')),
+                    new Choice('st_change_id_no', $('Change ID no.')),
                     new Choice('st_change_sanc', $('Change SANC no.')),
                     new Choice('st_change_persal', $('Change Persal no.')),
                     new Choice('isl_check_optout_optout', $('Stop SMS')),
                 ],
+                characters_per_page: 140,
+                options_per_page: 4,
```

Comment:
Set `options_per_page` to `null` and just comment it out since it defaults to `null`.  I like to leave it in the code as a comment so it's easy to change later.  When it's `null` the State calculates how many options to display, which is what we want.